src/App/AdminController.py

```diff
@@ -0,0 +1,492 @@
+    item_service,
+    jwt_service,
+    menu_service,
+    order_service,
+)
+from .JWTBearer import AdminBearer
+
+admin_router = APIRouter(prefix="/admin", tags=["Admin"], dependencies=[Depends(AdminBearer())])
+
+
+def get_admin_id_from_token(
+    credentials: Annotated[HTTPAuthorizationCredentials, Depends(AdminBearer())],
+) -> int:
+    token = credentials.credentials
+    customer_id = int(jwt_service.validate_user_jwt(token))
+    return customer_id
+
+
+# MENU
+@admin_router.get(
+    "/orderables", status_code=status.HTTP_200_OK, dependencies=[Depends(AdminBearer())]
+)
+def get_all_orderables(in_menu: bool = False):
+    try:
+        return menu_service.get_all_orderables(in_menu=in_menu)
+
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Error fetching orderables: {e}") from e
+
+
+@admin_router.post(
+    "/orderables/{orderable_id}/add",
+    status_code=status.HTTP_200_OK,
+    dependencies=[Depends(AdminBearer())],
+)
+def add_orderable_to_menu(orderable_id: int):
+    try:
+        menu_service.add_orderable_to_menu(orderable_id)
+        return f"The item with Orderable ID {orderable_id} has been added to the menu."
+    except ValueError as e:
+        raise HTTPException(
+            status_code=400, detail=f"[AdminController] Cannot add orderable to menu : {str(e)}"
+        ) from e
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Error fetching orderables: {str(e)}") from e
+
+
+@admin_router.delete(
+    "/orderables/{orderable_id}/remove",
+    status_code=status.HTTP_200_OK,
+    dependencies=[Depends(AdminBearer())],
+)
+def remove_orderable_from_menu(orderable_id: int):
+    try:
+        menu_service.remove_orderable_from_menu(orderable_id)
+        return f"The item with Orderable ID {orderable_id} has been removed from the menu."
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Error fetching orderables: {e}") from e
+    except ValueError as e:
+        raise HTTPException(
+            status_code=400,
+            detail=f"[AdminController] Cannot remove orderable from menu : {str(e)}",
+        ) from e
+
+
+# ITEMS
+@admin_router.get(
+    "/items/{item_id}", status_code=status.HTTP_200_OK, dependencies=[Depends(AdminBearer())]
+)
+def get_item_by_id(item_id: int):
+    try:
+        item = item_service.get_item_by_id(item_id)
+        if item is None:
+            raise HTTPException(status_code=404, detail=f"Item with id [{item_id}] not found")
+        return item
+    except Exception as e:
+        raise HTTPException(status_code=400, detail=f"Invalid request: {e}") from e
+
+
+file_bundle_image = File(None, media_type="image/*")
+
+
+@admin_router.post(
+    "/items", status_code=status.HTTP_201_CREATED, dependencies=[Depends(AdminBearer())]
+)
+async def create_item(
+    item_name: str,
+    item_price: float,
+    item_type: Literal["Starter", "Main course", "Dessert", "Side dish", "Drink"],
```

Comment:
This is probably a type that you use in different places in the code. Why not declare it in Model?